cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -53,10 +55,34 @@ func cleanupContainerDiskDirectory(ephemeralDiskDir string) {
 	}
 }
 
+func cleanupSerialConsoleSock(uid string) {
```

Comment:
I think it's safer if you do the opposite to send a signal to virt-tail which is first you create the file on virt-launcher startup and then you remove it to signal termination.

I know it's a detail what about Node disk pressure? If you're really really unlucky you might not be able to create the file and block termination.

Reply:
Good point, but I have a concern about this: 
by design the guest-console-log container mounts the private Kubevirt volume in ReadOnly mode (see:
https://github.com/kubevirt/kubevirt/pull/10110/files#diff-da9a4ec4b1d6fa8c0886cf67a4a02de97f2776727d670ccb73ed36ef31ea9136R33 ) to minimize its security footprint.
If we want to have virt-tail being able to create a file there, it will have to mount the volume with write permissions.

To be as safest as possible, virt-tail it's going to terminate itself on 3 different conditions:
1. sigterm
2. the unix socket file used for the serial console port disappearing
3. a termination file generated by virt-launcher-monitor if previous two mechanisms failed

As pointed out, a really high node disk pressure could prevent `virt-launcher-monitor` daemon from being able to generate the termination file. On the other side it should not affect the termination of the unix socket file used for the serial console port so the guest-console-log container still has the opportunity to terminate itself due to that.

Reply:
As discussed offline I meant creating the file upon virt-launcher-monitor startup and removing it upon termination.

Reply:
OK, done